.github/workflows/ci.yml

```diff
@@ -31,3 +31,12 @@ jobs:
         uses: codecov/codecov-action@v5
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
+
+      - name: Publish to staging repository
+        run: ./gradlew publishToSonatype currentVersion
```

Comment:
I can confirm that snapshot was uploaded: https://oss.sonatype.org/service/local/repositories/snapshots/content/pl/allegro/tech/build/axion-release/pl.allegro.tech.build.axion-release.gradle.plugin/1.18.16-regex-in-release-branch-names-SNAPSHOT/pl.allegro.tech.build.axion-release.gradle.plugin-1.18.16-regex-in-release-branch-names-20241125.063939-2.pom